assets/dev/js/frontend/handlers/base-nested-tabs.js

```diff
@@ -240,8 +240,8 @@ export default class BaseNestedTabs extends Base {
 		this.activateDefaultTab();
 	}
 
-	onEditSettingsChange( propertyName, value ) {
-		if ( 'activeItemIndex' === propertyName ) {
+	onEditSettingsChange( propertyName, value, route = '' ) {
+		if ( 'activeItemIndex' === propertyName && 'content' !== route ) {
```

Comment:
I noticed one problem, with my code: currently the active tabs don't change, when we select a repeater tab control.
I will study this behaviour further in the morning.